apps/discord/app.py

```
@@ -0,0 +1,344 @@
+def test_fetch_api():
+    """Test the API wrapper."""
+    result = fetch_api.remote()
+    if result.startswith("# 🤖: Oops! "):  # type: ignore
+        raise Exception(result)
+    else:
+        print(result)
+
+
+@app.function()
+@modal.concurrent(max_inputs=1000)
+async def reply(app_id: str, interaction_token: str):
```

Ruff (0.12.2)

Missing return type annotation for public function `reply`

Add return type annotation: `None`

(ANN201)